modules/ossm-security-mtls.adoc

```diff
@@ -23,10 +23,26 @@ kind: ServiceMeshControlPlane
 spec:
   version: v2.0
   security:
-    controlPlane:
+    dataPlane:
       mtls: true
 ----
 
+You can also set enable mTLS in the {product-title} web console.
```

Comment:
set enable -> (pick one)

Not critical, but wondering whether "by using" would be better than "in".